core/tests/protractor_utils/users.js

```diff
@@ -33,8 +35,21 @@ var login = async function(email, isSuperAdmin = false) {
   await (await driver.findElement(protractor.By.name('email'))).sendKeys(email);
   if (isSuperAdmin) {
     await (await driver.findElement(protractor.By.name('admin'))).click();
+    let adminCheckboxStatus = await driver.findElement(
+      protractor.By.name('admin')).getAttribute('checked');
+    expect(adminCheckboxStatus).toBeTruthy();
   }
   await (await driver.findElement(protractor.By.id('submit-login'))).click();
+  // The statement below uses a browser.wait() to determine if the user has
+  // logged in. Use of waitFor is not possible because the active page is
+  // non-angular.
```

Comment:
Can you double-check that waitFor doesn't work? [This StackOverflow answer](https://stackoverflow.com/a/44073142) recommends using `ExepectedConditions` for non-angular pages, which is what `waitFor.js` does

Reply:
Hi @U8NWXD, using expected conditions in the manner described [here](https://www.protractortest.org/#/api?view=ProtractorExpectedConditions.prototype.textToBePresentInElement) but it yields the following error:
```
Failed: Error while waiting for Protractor to sync with the page: "both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor's bootstrapping.  See http://git.io/v4gXM for details"
```

Do let me know if you have any specific thing you would like me to try.

Reply:
Okay, it's good to know that ExpectedCondition doesn't work in this case